File: atcoder_326_A.py
x, y = map(int, input().split())

if x<y:
    if abs(x-y) <= 2:
        print("Yes")
    else:
        print("No")
elif x > y:
    if abs(x-y) <= 3:
        print("Yes")
    else:
        print("No")

File: atcoder_326_A.cpp
#include<bits/stdc++.h>

using namespace std;

int main() {
    int x, y;
    cin >> x >> y;

    if (x < y) {
        if (abs(x - y) <= 2) {
            cout << "Yes" << endl;
        } else {
            cout << "No" << endl;
        }
    } else if (x > y) {
        if (abs(x - y) <= 3) {
            cout << "Yes" << endl;
        } else {
            cout << "No" << endl;
        }
    }

    return 0;
}

